doc/doxygen/src/roadmap.md

```diff
@@ -23,6 +23,11 @@ The text and items below are tentative, up for discussion, to be updated by regu
 - Point-to-Point Protocol (PPP): finalize and merge `gnrc_ppp`
 
 
+# Integrations
+(contact/steering: [Teufelchen](https://github.com/teufelchen1))
+
+- [Home-Assistant](https://www.home-assistant.io/) BTHome integration
```

Comment:
Something I wondered during the VMA, but didn't want derail too much (and I frankly have no idea what I am talking about): Could ESPhome also be on the list?

Reply:
Feel free to open a PR. I have to admit that I never lookes into ESPhome and rather considered it as a firmware for ESP/IDF devices for connecting them to Home Assistant.